src/cpp.cc

```diff
@@ -55,27 +55,31 @@ static std::string make_comments(const std::string &s, const std::string &c) {
   boost::char_separator<char> nl("\n", "", boost::keep_empty_tokens);
   boost::tokenizer<boost::char_separator<char> > lines(s, nl);
   for (boost::tokenizer<boost::char_separator<char> >::iterator i =
-       lines.begin(); i != lines.end(); ++i)
-    o << c << " " << *i << '\n';
+       lines.begin(); i != lines.end(); ++i) {
```

Comment:
yes: by convention, code lines should not exceed 80 characters - because old terminals had a width of 80 chars and tools like diff are easier to use on short lines